### PR TITLE
Boiler Plate, Database seeding file & Testing boiler

### PR DESCRIPTION
**Pull request by alexcravalho**
I added Jest and Enzyme then created a generic testing boiler that will check to see if my app will render successfully. This is mainly to ensure that Enzyme was setup correctly not to actually test the app.

### CODE REVIEW

File: client/src/components/App.test.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {shallow, mount} from 'enzyme';
+import App from './App.jsx';
+
+
+it('App renders without crashing', () => {
+ shallow(<App />);
+});
```
Comment (zz00024) [inline client/src/components/App.test.js]: Glad to see it works. I am thinking that it could be better to have more tests for more edge test cases.

Comment (zz00024) [reply client/src/components/App.test.js]: Other than that, I think this is good to go merge
